tables/en.tbl

```diff
@@ -1,4 +1,5 @@
-#+locale:en
+#+locale:en-US
+#+abae
```

Comment:
Can you have zero arity meta data?

And of course again why meta data change in tables other than Norwegian in this PR?

Reply:
> Can you have zero arity meta data?

Yes